src/docs/get-started/install/_android-sdk-setup.md

```diff
@@ -0,0 +1,46 @@
+## Android setup (without Android Studio)
+
+### Install Java
+
+```terminal
+$ sudo apt update
+$ sudo apt install default-jre
+$ sudo apt install default-jdk
+```
+
+### Install the Android SDK's
+
+Download the [Android SDK tools](https://developer.android.com/studio/#downloads) and select the “Command Line Tools only” option.
```

Comment:
We use a macro for this sort of link. In this case, it would be:

({{site.android-dev}}/studio/#downloads)